doom-modeline-core.el

```diff
@@ -789,7 +789,8 @@ then this function does nothing."
   "Cache the font width."
   (let ((attributes (face-all-attributes 'mode-line)))
     (or (cdr (assoc attributes doom-modeline--font-width-cache))
-        (let ((width (window-font-width nil 'mode-line)))
+        (let ((width (if (server-running-p) (window-width)
```

Comment:
Should check if `server-running-p` is loaded and valid.

Reply:
> Should check if server-running-p is loaded and valid.

You mean as in using fboundp?
```suggestion
        (let ((width (if (fboundp 'server-running-p) (window-width)
```
>  And why using (window-width) here for font width?

As i've written in this issue #354, by pure experimentation i found out that the former works with emacs daemon, while the latter (`window-font-width`) with the standalone instance. Don't want to cargo cult here but i really couldn't find an explanation other than works without issues